### ISSUE TITLE
Install.sh message is misleading

### ISSUE THREAD

**Issue by ln2max** [lncm/noma#62]
https://github.com/lncm/noma/blob/638db12b23f5413e27d9bb9fcc97b749d9906c58/install.sh#L143
```
$ git clone https://github.com/lncm/noma.git
$ cd noma/
$ sudo ./install.sh
Sources not found, fetching from github...
```
"But I just got the sources!"
Message should be something like
`Sources not found in /media, fetching from github...`

**Comment by nolim1t** [comment]
I have to agree with that. I think ```./install.sh``` should be worded a bit better or maybe make use of the current directory it is installed in?

**Comment by nolim1t** [comment]
For now the correct place it should really live in is ```/media/noma```  so you should ideally do a ```git clone https://github.com/lncm/noma.git /media```

**Comment by ln2max** [comment]
Add that to the README.md?

**Comment by nolim1t** [comment]
Good idea or maybe change install.sh

thoughts @AnotherDroog ?

**Comment by ln2max** [comment]
The relevant changes to README.md ended up getting merged as part of PR #69 ... https://github.com/lncm/noma/commit/e35c9f5995b305f52e38ff5df66f8eec11661744

Leaving the issue open in case @AnotherDroog has thoughts or someone wants to change install.sh
